src/router/test/Homepage.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import configureStore from '../../redux/configureStore';
+import '@testing-library/jest-dom/extend-expect';
+import Homepage from '../Homepage';
+
+describe('Tests the home page', () => {
+  const component = (
+    <Provider store={configureStore}>
+      <Homepage />
+    </Provider>
+  );
+
+  test('Match the snapshot', () => {
+    const tree = render(component);
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
- Your test fails here too. kindly fix it.

![image](https://github.com/cancelei/Stock-Exchange-APP/assets/59397159/6db4754c-60fb-44ae-b14b-c916f19fc6fe)